spec/convertion_by_format/check_open_docx_by_screen_spec.rb

```diff
@@ -5,8 +5,10 @@
 palladium = PalladiumHelper.new(DocumentServerHelper.get_version, 'Convert DOCX')
 result_sets = palladium.get_result_sets(StaticData::POSITIVE_STATUSES)
 files = s3.get_files_by_prefix('docx')
-image_size = nil
 describe 'Convert docx files by convert service' do
+  before(:each) do
```

Comment:
Leave one empty line after `before`, see https://rspec.rubystyle.guide/#empty-line-after-let
You don't need to use `(:each)`  in this line, see https://rspec.rubystyle.guide/#redundant-beforeeach